Components/list/list.css

```diff
@@ -0,0 +1,32 @@
+.list {
+  padding: 1rem;
+  width: 4rem;
+}
+
+.list li {
+  padding: 0.3rem;
+}
+
+.inline-list {
+  display: flex;
```

Comment:
suggestion:- a flex helper class can be used here , because If you use it then you don't have to write display:flex eyerywhere

Reply:
okay I will do that

Reply:
I have not added in documentation file yet , i will do that later

Reply:
done with the suggested changes , thanks for the review